### DISCUSSION TITLE
How to deploy an output standalone Next.js application in a PNPM workspace with Docker

### DISCUSSION THREAD

**Discussion by tunguyen-ct** [vercel/next.js#48604] [Help]
### Summary

Everything is fine until I have to decide which files/folders to copy to Docker image. The problem is copying the standalone folder is not enough, some dependencies are symlinked to the root node_modules.

**Comment by ImSingee** [comment]
Copy these files


- ./pkg/xxx/.next/standalone/*       ->     /app/pkg/xxx/*
- ./pkg/xxx/public/*                         ->     /app/pkg/xxx/pkg/xxx/public/*  
- ./pkg/xxx/.next/.next/static/*       ->     /app/pkg/xxx/pkg/xxx/.next/static/*

And then run `node /app/pkg/xxx/pkg/xxx/server.js`

**Reply by tbouasli** [reply]
node_modules isnt copied properly this way, because of the symlinks
